File: tourneyClient/src/models/entities/Characters/Ken.ts
import { CharacterId } from "@/models/Enums/CharacterId";
import { Character } from "../Character"
import { CharacterName } from "@/models/Enums/CharacterName";
import { Archetype } from "@/models/Enums/Archetype";
import { FallSpeed } from "@/models/Enums/FallSpeed";
import { WeightClass } from "@/models/Enums/WeightClass";
import { TierPlacement } from "@/models/Enums/TierPlacement";

export const Ken: Character =
{
    id: CharacterId.Ken,
    characterName: CharacterName.KEN,
    archetype: Archetype.BAIT_AND_PUNISH,
    fallSpeed: FallSpeed.FAST_FALLERS,
    weightClass: WeightClass.MIDDLEWEIGHT,
    tierPlacement: TierPlacement.B
}

export default Ken; 

File: tourneyAPI/Models/Entities/Characters/Ken.cs
namespace Entities;

using Enums;


// Defines the competitive profile metadata for this playable character.
public class Ken : Character
{
    // Initializes this character's default competitive attributes for matchmaking and tier logic.
    public Ken()
    {
        Id = CharacterId.Ken;
        characterName = CharacterName.KEN;
        archetype = Archetype.BAIT_AND_PUNISH;
        fallSpeed = FallSpeed.FAST_FALLERS;
        weightClass = WeightClass.MIDDLEWEIGHT;
        tierPlacement = TierPlacement.B;
    }

}